### PR TITLE
Make index.html a valid HTML page

### PR DESCRIPTION
**Pull request by farleyknight**

### CODE REVIEW

**Comment by farleyknight** [comment]
Lets have a conversation!

**Comment by farleyknight** [comment]
hey!

**Comment by setonpariser** [comment]
HI

File: index.html [hunk 1]
```diff
@@ -1,3 +1,11 @@
-Hello world
+<html>
+  <head>
+    <title>Seton's Github Pages</title>
```
Comment (farleyknight) [inline index.html:3]: Do you like this title?

Comment (setonpariser) [inline index.html:3]: Nope

**Pull request merged**